packages/repository/src/__tests__/unit/repositories/constraint-utils.unit.ts

```diff
@@ -77,6 +78,23 @@ describe('constraint utility functions', () => {
     });
   });
 
+  context('constrainWhereOr', () => {
+    const inputWhere: Where<{x: string; y: string; id: string}> = {
+      x: 'x',
+    };
+    it('enforces a constraint', () => {
+      const constraint = [{id: '5'}, {y: 'y'}];
+      const result = constrainWhereOr(inputWhere, constraint);
+      expect(result).to.deepEqual({...inputWhere, or: constraint});
+    });
+
+    it('enforces constraint with dup key', () => {
+      const constraint = [{y: 'z'}];
+      const result = constrainWhereOr(inputWhere, constraint);
```

Comment:
I am little bit confused. The test name says "with dup key", but IIUC, the code is using keys `x` and `y`. Where is the duplicated key?

Reply:
@bajtos My bad, forgot to add the key back. Fixed.